src/controllers/authenticationController.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from 'express';
+import { AuthenticationRepository } from '../repositories/authenticationRepository';
+import { BaseController } from './baseController';
+
+export class AuthenticationController extends BaseController {
```

Comment:
we don't need a specific controller, move the validate method to BaseController in order to be used in private actions.